glusterd2/commands/volumes/volume-create.go

```diff
@@ -29,7 +43,11 @@ func validateVolCreateReq(req *api.VolCreateReq) error {
 		return errors.New("invalid transport. Supported values: tcp or rdma")
 	}
 
-	if len(req.Subvols) <= 0 {
+	if req.Size > 0 && req.Size < minVolumeSize {
+		return errors.New("invalid Volume Size, Minimum size required is " + strconv.Itoa(minVolumeSize))
```

Comment:
suggestion: can you convert the capital case to a smaller case in the error message, wherever its required

Reply:
ok